Doc/library/sys.rst

```diff
@@ -1414,7 +1414,7 @@ always available.
      On Windows, UTF-8 is used for the console device.  Non-character
      devices such as disk files and pipes use the system locale
      encoding (i.e. the ANSI codepage).  Non-console character
-     devices such as NUL (i.e. where isatty() returns True) use the
+     devices such as NUL (i.e. where ``isatty()`` returns ``True``) use the
```

Comment:
Converts ``os.isatty()`` into an inline link:
```suggestion
     devices such as NUL (i.e. where :func:`os.isatty` returns ``True``) use the
```

Reply:
`isatty()` is not `os.isatty()`, but a method of the corresponding stream.